ci.sh

```diff
@@ -23,32 +23,44 @@ function run_tests {
   fi
 
   ./miri test --locked
-  if ! [ -n "${MIRI_TEST_TARGET+exists}" ]; then
+  if [ -z "${MIRI_TEST_TARGET+exists}" ]; then
     # Only for host architecture: tests with MIR optimizations
     #FIXME: Only testing opt level 1 due to <https://github.com/rust-lang/rust/issues/77564>.
     MIRIFLAGS="-Z mir-opt-level=1" ./miri test --locked
   fi
+
+  # On Windows, there is always "python", not "python3" or "python2".
+  if command -v python3; then
```

Comment:
```suggestion
  if command -v python3 >/dev/null; then
```
Let's silence its output